image_set_group

```diff
@@ -100,10 +100,12 @@ cp "${SRC_IMAGE}" "${DST_IMAGE}"
 UPDATE_PREFIX="${COREOS_PRODUCTION_IMAGE_NAME%_image.bin}_update"
 PRODUCTION_PREFIX="${COREOS_PRODUCTION_IMAGE_NAME%.bin}"
 CONTAINER_PREFIX="${COREOS_DEVELOPER_CONTAINER_NAME%.bin}"
+PCR_PREFIX="${COREOS_PRODUCTION_IMAGE_NAME%.bin}_pcr_policy"
```

Comment:
Since this isn't used multiple times with different suffixes how about just PCR_POLICY and include the .zip